dynawo/sources/Models/Modelica/Dynawo/Examples/Generators/System1Model.mo

```diff
@@ -0,0 +1,82 @@
+  connect(Line1.terminal2, infiniteBus1.terminal) annotation(
+    Line(points = {{26, 0}, {40, 0}, {40, 40}}, color = {0, 0, 255}));
+  connect(Line1bis.terminal2, infiniteBus1.terminal) annotation(
+    Line(points = {{26, 80}, {40, 80}, {40, 40}}, color = {0, 0, 255}));
+  connect(Generator2.terminal, Line1.terminal1) annotation(
+    Line(points = {{-40, 20}, {6, 20}, {6, 0}}, color = {0, 0, 255}));
+  connect(Generator2.terminal, Line1bis.terminal1) annotation(
+    Line(points = {{-40, 20}, {6, 20}, {6, 80}}, color = {0, 0, 255}));
+  connect(Generator1.terminal, Line1bis.terminal1) annotation(
+    Line(points = {{-40, 60}, {6, 60}, {6, 80}}, color = {0, 0, 255}));
+  connect(Generator1.terminal, Line1.terminal1) annotation(
+    Line(points = {{-40, 60}, {6, 60}, {6, 0}}, color = {0, 0, 255}));
+  annotation(
+    experiment(StartTime = 0, StopTime = 3000, Tolerance = 1e-06, Interval = 10),
+    __OpenModelica_commandLineOptions = "--matchingAlgorithm=PFPlusExt --indexReductionMethod=dynamicStateSelection -d=initialization,NLSanalyticJacobian --daeMode",
+    __OpenModelica_simulationFlags(ls = "klu", lv = "LOG_STATS", nls = "kinsol", s = "euler"),
+  Documentation(info = "<html><head></head><body><div><div><div>This test case describes the disconnection of Line1bis from the infinite bus caused by an event. Line1 and Line1bis are connected to an infinite bus and two generators PQ with a proportional reactive power regulation.</div><div><br></div><div>Both generator models calculate the active power PGenPu controlled by the signal N for primary frequency regulation and the reactive power QGenPu controlled by the signal NQ for primary voltage regulation. The values of PGenPu are within [PMinPu, PMaxPu], and the values of QGenPu are within [QMinPu, QMaxPu]. In the GeneratorPQProp model (used for Generator 1), QMinPu and QMaxPu are given. In the GeneratorPQPropdiagramPQ model (used for Generator 2), QMinPu and QMaxPu are calculated by referring to the PQ diagram while knowing the value of PGenPu.</div><div><br></div><div>At t = 50s, Line1bis is disconnected from the infinite bus; thus, the voltage at the generator level will increase. Signal NQ connected to both generators is calculated with a PI controller and will be different than 0 since URegulated is greater than URef. As a consequence, the reactive power generation QGenPu of the generators will decrease; thus, the URegulated will decrease following the PI control.</div><div>The freezing should be activated to stop the integration when the reactive power limits are reached.</div><div><br></div><div>At t = 70s, SignalN modeled as a step function will modify the generated active power PGenPu of generator 1 and generator 2 as long as it stays within the minimum and maximum limits. Here, PGenPu will be increased, causing the URegulated at the generator level to decrease below URef. Signal NQ will then increase the voltage.</div></div></div></body></html>"));
```

Comment:
Maybe we can first describe the test case : two generators, one infinite bus, two lines. The generators use a coordinated voltage control (SignalNQ) that dispatches the needed reactive power between the two generators to control the voltage at the generators connection bus. We simulate the disconnection of a line, bla bla.
We can also say here that SignalN is simply modelled by a step and not a proper regulation, as the infinite bus is here to balance the generation and consumption. Here N is simply used to model an active power reference step.

SignalNQ does not increase the voltage directly, it increases Q, which then results in an increase in the voltage.